ParcialProgramacion4/src/Empresa.java

```diff
@@ -58,4 +60,371 @@ private Habilidad buscarHabilidad(String nombre) {
     	return null;
     }
     
+
+    //CASO DE USO AGREGAR EMPLEADO AL SISTEMA
+    public void agregarEmpleado(Scanner scanner) {
+        System.out.println("Numero de legajo: ");
+        int legajo = Integer.parseInt(scanner.nextLine());
+
+        Empleado empleadoRepetido = this.buscarEmpleado(legajo);
+
+        if(empleadoRepetido == null) {
+            System.out.println("Nombre: ");
```

Comment:
Es una boludes, pero usa print asi te deja ingresar la entrada al lado del texto
```suggestion
            System.out.print("Nombre: ");
```